service/controller/machinepool/handler/migration/testdata/case-0-New-MachinePool-references-not-updated_MachinePool.golden

```diff
@@ -0,0 +1,60 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
New MachinePool CR that just got created, but the migration is not complete (CR references are not updated). Handler does not do anything, it waits for the references to be updated, so here we see that the new MachinePool CR is not changed by the new MachinePool migration handler.